src/morello-install/README.md

```diff
@@ -558,35 +542,78 @@ Then press Enter to continue.
 
 #### Add user accounts
 
+If you plan to install the desktop environment in the next step, it is
+useful to create user accounts now.
 If desired, select Yes and press Enter to add non-root accounts.
 Otherwise, select No and press Enter
 
+#### CHERI Desktop environment
+
+If you wish to configure your CheriBSD system with a KDE-based desktop
+environment you can do so now.  **NOTE: you just have previously
+configured the network and be able to reach pkg.cheribsd.org.**
```

Comment:
It's quite hard to read. Maybe:
```
**NOTE: You have had to previously configure the network and you must be able to reach pkg.CheriBSD.org.
```
```

Reply:
It would be the best if the installer could allow to go back to the network configuration, or even better not display this step if network isn't configured. Maybe the network setup step could advise a user to configure it so that they can install additional third-party software in a later step.

Reply:
A one-letter typo rendered that unreadable.

I think the installer does allow that, but we won't diagnose network failures usefully, just fail the install. Given the infinite set of possible middle box misconfigurations I'm not sure there's much we can do there.